### PR TITLE
Added WAF Notebook

### PR DESCRIPTION
**Pull request by petebryan**
Added Azure Front Door WAF SQLI event triage notebook that allows users to triage SQLI events and apply rule exclusions to WAF rulesets.

### CODE REVIEW

**Comment by sowmyam2019** [comment]
Please add some details around the Threat Intelligence section describing how the SQL injection alert entity data is matched against TI.

**Comment by sowmyam2019** [comment]
In the last cell update the comment to include the following " Incident triaged in notebook, WAF policy updated with exclusions". Need to change WAF rule to WAF policy.

File: Guided Investigation - Azure WAF SQLI.ipynb [hunk 1]
```diff
@@ -0,0 +1,792 @@
+{
```
Comment (ianhelle) [inline Guided Investigation - Azure WAF SQLI.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #19.        sqli_analytics = analytics[(analytics['properties.query'].str.contains("FrontDoorWebApplicationFirewallLog") | analytics['properties.query'].str.contains("ApplicationGatewayFirewallLog")) &amp; (analytics['properties.query'].str.contains("SQLI") | analytics['properties.query'].str.contains("SQL Injection"))]
</pre><p>I thought you needed parentheses around every sub-expression if joined with | or &amp;. </p><p>The first 2 or'd expressions don't have that.</p><p>Might also be easier to read if split on multiiple lines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (ianhelle) [inline Guided Investigation - Azure WAF SQLI.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #23.            qry = f"SecurityAlert | where TimeGenerated between((datetime({start})-7d)..datetime({end})) | where SystemAlertId == '{alert['ID']}'"
</pre><p>Does nested 'ID' string work? in "where SystemAlertId == '{alert['ID']}'" - seems like it shouldn't</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (ianhelle) [inline Guided Investigation - Azure WAF SQLI.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #16.        # ToDo add support if not an OWASP rule
</pre><p>The "todo" is done right? in the else clause?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (ianhelle) [inline Guided Investigation - Azure WAF SQLI.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #43.    if isinstance(rule_details_df, pd.DataFrame) and not rule_details_df.empty:
</pre><p>I think I created a utils function to do this (for future ref) msticpy.common.data_utils.df_has_data</p><p>Although the import may be longer to type than the saving if you only have one check!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (petebryan) [reply Guided Investigation - Azure WAF SQLI.ipynb:1]: It does work thankfully!

Comment (petebryan) [reply Guided Investigation - Azure WAF SQLI.ipynb:1]: Removed, thanks!

Comment (petebryan) [reply Guided Investigation - Azure WAF SQLI.ipynb:1]: Did some extra testing and it appear to work as intended - I've split over 2 lines to improve readability.

**Pull request merged**